cpp/ql/src/codeql-suites/slow-queries.yml

```diff
@@ -0,0 +1,11 @@
+- description: C/C++ queries which are infeasible to compute on large projects
```

Comment:
I think `slow-queries.yml` is a misleading name. This suite excludes slow queries, not include them. I think `exclude-slow-queries.yml` would be clearer.

Reply:
@jbj Could these queries be optimised, so they don't need to be excluded? The majority seem related to unclosed resources.

Reply:
The reason they're not scalable is that they use the old Steensgaard-style pointer analysis. Optimising them would mean rewriting with modern techniques -- which is a worthy goal, of course, but not an easy one without affecting results.